src/CustomActions.js

```diff
@@ -39,24 +39,24 @@ Actions.replaceAction("HoldCall", (payload, original) => {
 
 Actions.replaceAction("UnholdCall", (payload, original) => {
   return new Promise((resolve, reject) => {
-
     const task = payload.task;
-    if (Number(task.age) <= 2) {
-      original(payload)
-    } else {
-      const reservation = payload.task.sourceObject;
-      const conference = task.attributes.conference.sid;
-      const participant = task.attributes.conference.participants.customer;
-      const hold = false;
+    const reservation = task.sourceObject;
+    const { conference } = task.attributes;
 
-      if (task.taskChannelUniqueName === "voice" && reservation.task.attributes.direction === "outbound") {
-        toggleHold(conference, participant, hold, original, payload, reservation);
-      } else {
-        original(payload);
-      }
+    if (!conference || Object.keys(conference.participants) === 0) {
+      // Do nothing if there is no one in the call.
+      return;
     }
+    const { customer } = conference.participants;
+
+    if (task.taskChannelUniqueName === "voice" && task.attributes.direction === "outbound") {
```

Comment:
I think that the name changed [here](https://github.com/lehel-twilio/plugin-dialpad/commit/4ceba96501277ef0ac023a80491fbf8b2ecb15f1#diff-a112a41159080ae877705c85082b0131R26) to `custom1`.

Reply:
I think, it is broken on `master` as well because as part of the change, the hold and unhold actions were not updated. I'll update my PR.

cc @lehel-twilio